parm/stage/atmosphere_cold_RT.yaml.j2

```
@@ -3,7 +3,7 @@ atmosphere_cold_RT:
         - "{{ COMOUT_ATMOS_RESTART_PREV_MEM }}"
         - "{{ COMOUT_ATMOS_ANALYSIS_MEM }}"
         - "{{ COMOUT_ATMOS_HISTORY_MEM }}"
-    copy:
+    link:
```

```suggestion
    link_req:
```
Use `link_req` to ensure that it checks the file is in place.